src/lib/definitions.js

```diff
@@ -383,6 +383,9 @@ function getFHPaddingEntries(index)
     var LOCALE_INFINITY                 = Feature.LOCALE_INFINITY;
     var LOCALE_NUMERALS                 = Feature.LOCALE_NUMERALS;
     var LOCALE_NUMERALS_EXT             = Feature.LOCALE_NUMERALS_EXT;
+    var LOCALE_NUMERALS_IE11_WIN7       = Feature.LOCALE_NUMERALS_IE11_WIN7;
+    var LOCALE_NUMERALS_IE11_WIN7_8     = Feature.LOCALE_NUMERALS_IE11_WIN7_8;
+    var LOCALE_NUMERALS_IE11_WIN8       = Feature.LOCALE_NUMERALS_IE11_WIN8;
```

Comment:
I'm not sure if this division makes sense. It seems that these features depend on the version of certain DLLs used by IE, and these DLLs could be updated independently from the operating system, for example with Windows updates. This is especially true for Windows 7 and 8 where IE was integrated with other system components.

Reply:
so, what should I do? should I just remove Latvian and Russian related definition?

Reply:
I don't know yet. I will have to do some more analysis.

Reply:
is your analysis finished now?

Reply:
and, what should I do to Latvian and Russian related definition?